packages/identity-app/src/SavedAccounts.tsx

```diff
@@ -45,11 +51,22 @@ export class SavedAccounts extends React.PureComponent<Props> {
                     orientation='horizontal'
                     size='small'
                   />
+                  <Subscribe>
+                    {
+                      // FIXME using any because freeBalance gives a Codec here, not a Balance
```

Comment:
Yes, it is an issue - since these are all determined at runtime, there is no way (atm) to specify the actual type in the signature. You can just cast it as a Balance